pyaedt/modeler/Primitives3DLayout.py

```diff
@@ -1043,3 +1043,41 @@ def place_3d_component(
         comp = ComponentsSubCircuit3DLayout(self, comp_name.split(";")[-1])
         self.components_3d[comp_name.split(";")[-1]] = comp
         return comp  #
+
+    def create_text(self, text, position, angle=0, font_size=12):
+        """Creates a text primitive object.
+
+        Parameters
+        ----------
+        text : str
+            Text string.
+        position : list
+            Position of the text.
+        angle : str
```

Comment:
It might be a float here
```suggestion
        angle : float, optional
```